pkg/config/config.go

```diff
@@ -1042,6 +1042,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("admission_controller.inject_config.trace_agent_socket", "unix:///var/run/datadog/apm.socket")
 	config.BindEnvAndSetDefault("admission_controller.inject_config.dogstatsd_socket", "unix:///var/run/datadog/dsd.socket")
 	config.BindEnvAndSetDefault("admission_controller.inject_tags.enabled", true)
+	config.BindEnvAndSetDefault("admission_controller.inject_tags.workload_context", false)
```

Comment:
it can be useful create a section in `admission_controller` for `cws` for futur feature. 

```suggestion
	config.BindEnvAndSetDefault("admission_controller.cws.workload_context", false)
```